mozregression/cli.py

```diff
@@ -389,9 +392,16 @@ def validate(self):
             # this can be useful for both inbound and nightly, because we
             # can go to inbound from nightly.
             fetch_config.set_inbound_branch(options.inbound_branch)
+        # set action for just use changset or data to bisect
+        if options.launch:
+            try:
+                options.launch = parse_date(options.launch)
```

Comment:
Hmm, Some errors shows up when I use --launch 2014-12-15
1:35.07 LOG: MainThread ERROR HTTPSConnectionPool(host='archive.mozilla.org', port=443): Max retries exceeded with url: /pub/mozilla.org/firefox/nightly/2014/12/ (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x1036d9990>, 'Connection to archive.mozilla.org timed out. (connect timeout=30.0)'))

And I print the options.launch, it still shows '2014-12-15' rather than date(2014,12,15) which it may should be.